gapic/templates/%namespace/%name_%version/%sub/services/%service/transports/grpc.py.j2

```diff
@@ -133,8 +133,8 @@ class {{ service.name }}GrpcTransport({{ service.name }}Transport):
             warnings.warn("client_cert_source is deprecated", DeprecationWarning)
 
         if channel:
-            # Ignore credentials if a channel was passed.
-            credentials = False
+            # Ignore credentials if a channel was passed without credentials.
+            credentials = False if not credentials else credentials
```

Comment:
```suggestion
            credentials = credentials or False
```
if you really need a Boolean value, or even better, just delete the line entirely: if `credentials` is not passed in, it is `None` so it evaluates as `False`. In other words, if you don't need a strict Boolean type, this line is a no-op and can be removed. But again, it changes the behavior of this method.

Reply:
addressed!